db/migrations/20220920015521_option_selector.sql

```diff
@@ -0,0 +1,12 @@
+-- migrate:up
+CREATE TABLE option_selector(
```

Comment:
테이블 이름을 통해 중간 테이블이라는 것을 알게 하는 것이 좋을 것 같습니다.
```suggestion
CREATE TABLE option_items(
```